extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -39,10 +39,8 @@ export class AmpAutocomplete extends AMP.BaseElement {
 
   /** @override */
   buildCallback() {
-    if (!isExperimentOn(this.getWin(), 'amp-autocomplete')) {
-      user().warn('Experiment %s is not turned on.', EXPERIMENT);
-      return;
-    }
+    userAssert(isExperimentOn(this.win, 'amp-autocomplete'),
```

Comment:
Is there a meaningful difference between user().warn() and userAssert()? I learned why user().warn() was causing trouble in the Travis checks so we can actually undo this change, but I am curious if one way is preferable to the other.

Reply:
Good question: as I understand it, `userAssert()` logs red in the console and stops execution of the script beyond that point. `user().warn()` logs yellow in the console and execution continues.

Reply:
Cool, I'll bring it back to use `user().warn()` then because that is more in the spirit of the documentation 👍

Reply:
Ah sorry, I should have answered why we prefer the `user().error` in the case of experiments: we want the execution to stop if the user doesn't have the experiment enabled. This prevents people from using and depending on the extension of the API before we are ready to launch.